### PR TITLE
Pass values into function call

### PR DESCRIPTION
**Pull request by MichaelDimmitt**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
-(() => {
-  let rows = 12
+((rowNum, colNum, rowCondition, paddingNum, sizeNum) => {
```
Comment (reergymerej) [inline index.js:1]: My religious beliefs disallow any more than 3 parameters in a function.  :D 

They do allow for 
```
{ rowNum, colNum, ...}
```

File: index.js [hunk 2]
```diff
@@ -118,8 +118,8 @@
 
     const root = document.querySelector('#hive')
 
-    const size = 50
-    const padding = 2
+    const padding = paddingNum
```
Comment (reergymerej) [inline index.js:121]: Reassignment doesn't seem necessary.

File: index.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
   const hive = (() => {
     const grid = []
     while (rows--) {
       const row = []
-      let cols = 36
+      let cols = colNum
       while (cols--) {
         row.push(
-          Math.random() > 0.2
+          Math.random() > rowCondition
```
Comment (reergymerej) [inline index.js:11]: I like that this is no longer a magic number.  What name would better describe this?  `probabilityOfASuperCoolCell`?

File: index.js [hunk 4]
```diff
@@ -138,4 +138,4 @@
   }
 
   drawHive(hive)
-})()
+})(12, 36, 0.2, 0, 50)
```
Comment (reergymerej) [inline index.js:141]: Where's my decoder ring? 💍

**Comment by reergymerej** [comment]
Danke, @MichaelDimmitt!

**Comment by MichaelDimmitt** [comment]
@reergymerej, Did you see the other one? That builds on top of this pr and lets you have 3 different presets/versions.

**Comment by MichaelDimmitt** [comment]
Updated in the most recent pull request 🙂 #4

**Pull request closed without merging**